clar2wasm/src/wasm_generator.rs

```diff
@@ -2996,7 +3027,7 @@ impl WasmGenerator {
         for operand in operands {
             self.traverse_expr(builder, operand)?;
         }
-        Ok(())
+        self.visit_logical(builder)
```

Comment:
I know you didn't name these, but while you're making changes here, let's combine `traverse_logical` and `visit_logical` and rename to `traverse_logical_not`.

Reply:
Done!